diff -up arptables_jf-0.0.8/arptables.c.options arptables_jf-0.0.8/arptables.c
--- arptables_jf-0.0.8/arptables.c.options	2013-05-15 15:09:45.328799344 +0200
+++ arptables_jf-0.0.8/arptables.c	2013-05-15 15:10:21.028301662 +0200
@@ -192,7 +192,7 @@ static char commands_v_options[NUMBER_OF
 /*DELETE_NUM*/{'x','x',' ','x',' ','x','x','x','x','x','x','x','x','x','x','x'},
 /*REPLACE*/   {'x',' ',' ','x',' ','x',' ',' ',' ',' ',' ',' ',' ',' ',' ',' '},
 /*APPEND*/    {'x',' ',' ','x',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' '},
-/*LIST*/      {' ','x',' ',' ',' ','x','x','x','x','x','x','x','x','x','x','x'},
+/*LIST*/      {' ','x',' ',' ',' ',' ','x','x','x','x','x','x','x','x','x','x'},
 /*FLUSH*/     {'x','x',' ','x',' ','x','x','x','x','x','x','x','x','x','x','x'},
 /*ZERO*/      {'x','x',' ','x',' ','x','x','x','x','x','x','x','x','x','x','x'},
 /*NEW_CHAIN*/ {'x','x',' ','x',' ','x','x','x','x','x','x','x','x','x','x','x'},
