diff -up arptables_jf-0.0.8/arptables.c.mangle arptables_jf-0.0.8/arptables.c
--- arptables_jf-0.0.8/arptables.c.mangle	2010-03-25 11:18:07.351682117 +0100
+++ arptables_jf-0.0.8/arptables.c	2010-03-25 11:19:07.730682571 +0100
@@ -1978,6 +1978,8 @@ int do_commandarp(int argc, char *argv[]
 			check_inverse ( optarg, &invert, &optind, argc );
 			set_option ( &options, OPT_ARHLN, &fw.arp.invflags, invert );
 			arhln = argv[optind - 1];
+			if ( arhln )
+			    parse_uint8 ( arhln, &fw.arp.arhln, &fw.arp.arhln_mask );
 			break;
 
 		case 'p':
